mw-webapp/src/component/header/Header.tsx

```diff
@@ -18,17 +18,17 @@ export const Header = () => {
   const [user, setUser] = useState<User | null>(null);
 
   useEffect(() => {
-    handleUserAuthState(setUser);
+    userAuthState(setUser);
```

Comment:
functions should contain a verb because it should do something

Reply:
Add verb handle to the name fo the function